git_machete/cli.py

```diff
@@ -142,7 +142,7 @@ def create_cli_parser() -> argparse.ArgumentParser:
 
     subparsers.add_parser('format')
 
-    github_parser = subparsers.add_parser('github', default=argparse.SUPPRESS)
+    github_parser = subparsers.add_parser('github', argument_default=argparse.SUPPRESS)
```

Comment:
Wooow what's the difference here? 😯

Reply:
Top one doesn't work 😄 . 
`default` is a parameter of a `add_argument` method, whereas `argument_default` is an argument of a `add_parser` method. This change is basically a fix for an error I've made.